plio/tests.py

```diff
@@ -345,8 +345,8 @@ def test_user_list_items_by_plio(self):
         # should not be included
         self.assertEqual(len(response.data), 1)
 
-    def test_user_list_own_items(self):
-        """A user should only be able to list their own items"""
+    def test_user_list_other_items(self):
```

Comment:
```suggestion
    def test_user_list_other_user_items(self):
```
Just to stay consistent with the naming we've done for other test functions